### PR TITLE
[fix bug 1347842] Move event management from database to file.

### PR DESCRIPTION
**Pull request by glogiotatidis**
- Removes /admin and saml related code.
- Moves university event management to a YAML file.

### CODE REVIEW

File: university_events.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Store university events here. All fields are required.
+- name: Example
+  location: Imaginary place
+  start_date: 2017-01-20
+  end_date: 2017-01-22
```
Comment (glogiotatidis) [inline university_events.yml]: is this data schema ok @bensternthal ?

File: careers/university/tests/test_views.py [hunk 2]
```diff
@@ -27,20 +26,30 @@ def test_event_filter(self):
...
 
-        with patch('careers.university.views.date') as mock_date:
-            mock_date.today.return_value = date(2010, 1, 6)
-            response, context = self._index()
+            with patch('careers.university.views.date') as mock_date:
+                mock_date.today.return_value = date(2010, 1, 6)
+                response, context = self._index()
 
-        self.assertEqual(list(context['events']), [event4, event3, event1, event2])
+        self.assertEqual(list(context['events']), [event1,  event2, event3, event4])
```
Comment (glogiotatidis) [inline careers/university/tests/test_views.py]: Order is tests in `test_utils.py`

**Comment by glogiotatidis** [comment]
@jgmize r?

File: university_events.yml (hunk 1 above)
Comment (bensternthal) [inline university_events.yml]: R+

Comment (jgmize) [inline university_events.yml]: Are there any actual events that we need to populate in this file prior to merge?

Comment (glogiotatidis) [inline university_events.yml]: Nope, list is clear atm. good point though

**Comment by glogiotatidis** [comment]
Thanks both!

**Pull request merged**